assets/src/room_ui.ts

```diff
@@ -67,23 +67,26 @@ export function addVideoStatusChangedCallback(
 
 export function setMicIndicator(peer_id: string, active: boolean) {
   let image = document.querySelector(`#feed-${peer_id} img`);
+  let imageCam = document.querySelector(`#feed-${peer_id} img[name='camera']`);
+
 
   if (active) {
     image?.classList.add("invisible");
+    imageCam?.classList.remove("cam-margin")
```

Comment:
Does it really have to be removed if invisible?

Reply:
This code causes that when the microphone icon is invisible the camera icon is stuck to the left edge instead of staying in the center. If there are two, the camera moves away from the edge, and if only the camera, it takes the place of the microphone. I think it looks more aesthetic.